man/systemd-system.conf.xml

```diff
@@ -162,6 +162,30 @@
         These settings have no effect if a hardware watchdog is not available.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>PreTimeoutWatchdogSec=</varname></term>
```

Comment:
so as i take it the same concept could be applied to the reboot timeout too. hence, I think this one should be named RuntimeWatchdogPreSec=, i.e. to underline that this is closely related to RuntimeWatchdogSec=.